observability/src/main/proto/grpc/observabilitylog/v1/observabilitylog.proto

```diff
@@ -69,7 +69,7 @@ message GrpcLogRecord {
   // Each call may have several log entries. They will all have the same rpc_id.
   // Nothing is guaranteed about their value other than they are unique across
   // different RPCs in the same gRPC process.
-  uint64 rpc_id = 2;
+  string rpc_id = 2;
```

Comment:
CC @stanley-cheung @lidizheng when did we change rpc_id to `string`

Reply:
In a meeting (notes forwarded to you), we found that if RPC_ID is an integer counter might not be the most friendly way for searching. Multiple processes might have the same RPC_ID, making it harder to pull events about an RPC. So, after discussing with Stanley, we decided to change RPC_ID from an int counter to UUID.